src/sentry/options/store.py

```diff
@@ -87,10 +57,17 @@ def __init__(self, cache=None, ttl=None):
         self.ttl = ttl
         self.flush_local_cache()
 
-    @cached_property
+    @property
```

Comment:
Should this continue using `cached_property` or  do we need the model class selection code to run each time the property is accessed?

Reply:
Yes, mainly for tests.  Auditing it, in production this call won't be hit often due to the cache.